include/vrv/layerelement.h

```diff
@@ -576,6 +581,9 @@ class LayerElement : public Object,
      * This also stores the negative values for identifying cross-staff
      */
     int m_alignmentLayerN;
+
+    // flag to indicate that layerElement belongs to the beamSpan
+    bool m_isInBeamspan;
```

Comment:
This needs to be initialized (I don't think it is) and reset in `LayerElement::ResetDrawing`

Reply:
This change is basically moving from **public** to **private** section, it's already existing value and is being initialized.
https://github.com/rism-digital/verovio/blob/e59394fffd1b48f3c68e72c1a1396be91fcd3930/src/layerelement.cpp#L134

Also I don't think `LayerElement::ResetDrawing` even exists. If you meant `LayerElement::ResetHorizontalAlignment`, then I don't think it's the best idea to reset it there - notes should not change whether they are in the beamspan or not based on alignment.

Reply:
Sorry, I meant `LayerElement::ResetData` (which used to be called `ResetDrawing`). And I can see it is actually reset here, so we are all good.